### PR TITLE
fix panaroo conda env

### PR DESCRIPTION
**Pull request by kapsakcj**
unpins openssl build version in panaroo conda env. pinned version was labeled "broken" according to this: https://anaconda.org/conda-forge/openssl/files

This small change allows the conda/micromamba environment to solve and build successfully.

Pull Request (PR) checklist:
- [x] Include a description of what is in this pull request in this message.
- [x] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [x] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [x] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [x] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [x] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [x] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [x] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [x] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

**Comment by kapsakcj** [comment]
@kevinlibuit Please review & approve when you get a chance. Had to make this small change since the conda environment would not solve when Erin deployed it this morning. Error log here: https://github.com/StaPH-B/docker-builds/runs/7077952183?check_suite_focus=true

We should see the tests pass with this change 🤞

**Comment by kapsakcj** [comment]
✅

**Comment by erinyoung** [comment]
@kevinlibuit is assigned the reviewer, but I'd accept this and merge it.

**Comment by kapsakcj** [comment]
@erinyoung yes please! I will deploy once you give the approval

Kevin is a bit busy considering he has a BIG day coming up very soon 👶

**Review by erinyoung — APPROVED** [review]
Yay!

**Pull request merged**
